### PR TITLE
[QA-511]: Update VC script

### PR DESCRIPTION
**Pull request by abhinaysrivastavaperf**
### What?
Added script for update VC scenario

#### Changes:
- Modified the function for create VC script.
- Added a new function for update VC call.
- Added the script for update VC scenario.

---

### Why?
To conduct performance tests for update VC

### CODE REVIEW

File: deploy/scripts/src/accounts/vc-storage.ts [hunk 1]
```diff
@@ -140,22 +148,86 @@ export function summariseVC(): void {
...
 
-interface VCPayload {
+interface CreateVCPayload {
   vc: string
   state: string
   provenance: string
 }
 
-function generateVCPayload(vcJWT: string): VCPayload {
+function generateCreateVCPayload(vcJWT: string, vcState: string): CreateVCPayload {
```
Comment (Tom-Dann) [inline deploy/scripts/src/accounts/vc-storage.ts:215]:
```suggestion
function generateCreateVCPayload(vcJWT: string, vcState: VcState): CreateVCPayload {
```
```
type VcState = 'CURRENT' | 'PENDING'
```
Could make a type for the VC state to ensure it it always of the correct value

Comment (abhinaysrivastavaperf) [reply deploy/scripts/src/accounts/vc-storage.ts]: Updated

**Review by Tom-Dann — CHANGES_REQUESTED** [review]

File: deploy/scripts/src/accounts/vc-storage.ts [hunk 2]
```diff
@@ -95,7 +99,11 @@ export function persistVC(): void {
...
   }
 
-  const body = JSON.stringify(Object.values(jwtTokens).map(generateVCPayload))
+  const body = JSON.stringify([
+    generateCreateVCPayload(jwtTokens.fraud, 'PENDING'),
+    generateCreateVCPayload(jwtTokens.passport, 'PENDING'),
+    generateCreateVCPayload(jwtTokens.kbv, 'PENDING')
+  ])
```
[the remark this answers is not in the document]
Comment (abhinaysrivastavaperf) [reply deploy/scripts/src/accounts/vc-storage.ts]: Thanks. Updated now

File: deploy/scripts/src/accounts/vc-storage.ts [hunk 3]
```diff
@@ -140,22 +148,87 @@ export function summariseVC(): void {
...
+  }
+
+  body = JSON.stringify([
+    generateUpdateVCPayload(jwtSignatures.fraud, 'CURRENT'),
+    generateUpdateVCPayload(jwtSignatures.passport, 'CURRENT'),
+    generateUpdateVCPayload(jwtSignatures.kbv, 'CURRENT')
+  ])
+
+  timeGroup(groups[1], () => http.patch(env.envURL + `/vcs/${subjectID}`, body, options), {
+    isStatusCode202: r => r.status === 204,
```
[the remark this answers is not in the document]
Comment (abhinaysrivastavaperf) [reply deploy/scripts/src/accounts/vc-storage.ts]: Updated

**Review by Tom-Dann — APPROVED** [review]

**Pull request merged**